mindmaps-graql/src/main/java/io/mindmaps/graql/internal/analytics/Analytics.java

```diff
@@ -90,7 +88,8 @@ public Analytics(String keySpace) {
         excludedTypes.addAll(graph.getMetaRoleType().instances());
         excludedTypes.addAll(graph.getMetaRuleType().instances());
 
-        // collect analytics resource type to exclude
+        // collect analytics resource types to exclude
+        HashSet<String> analyticsElements = Sets.newHashSet(Analytics.degree, GraqlType.HAS_RESOURCE.getId(Analytics.degree));
```

Comment:
move resources from analytics into subgraph and implement user specified resources